examples/dummy_plugin/dummy_plugin/templates/dummy_plugin/inc/panel_site_overwrite.html

```diff
@@ -0,0 +1,3 @@
+<div class="list-group-item">
+    <a>Site has been overwritten.</a>
+</div>
```

Comment:
Hm. I'm actually not sure I like allowing plugins to *replace* home page items. Maybe this is something we should check for and block?

Reply:
okay sure, i will block this for now